SPECIFICATION.md

```diff
@@ -101,17 +101,23 @@ the number may be omitted. If the integer part is zero and there is a fractional
 part, the zero may be omitted (e.g. `-.5` instead of `-0.5`).
 
 The coordinate system starts at the lower left corner. Positive X coordinates
-point to the right. Positive Y coordinates point to the top. Negative
-coordinates are not allowed.
+point to the right. Positive Y coordinates point to the top. A value of 9 means
+100% of the font height (i.e. the coordinate system is scaled with factor 9).
```

Comment:
"Font height" is ambiguous. Maybe use the term "cap height"? See https://github.com/fontobene/fontobene/issues/5

We should probably also include that illustration in the spec. We have permission to do so.